chapters/CRUD-Operations.md

```diff
@@ -185,25 +216,37 @@ It returns the following output:
 The Deterministically Generate Beacon Services algorithm generates three
 ::Beacons:: from the single `keyBytes` value used to generate the deterministic
 **did:btc1**, one for each of the following three Bitcoin address types for the
-Bitcoin `network` specified by the DID: Pay-to-Public-Key-Hash (P2PKH),
-Pay-to-Witness-Public-Key-Hash (P2WPKH), and Pay-to-Taproot (P2TR). Spends from
-these three addresses can be produced only through signatures from the
-`keyBytes`'s associated private key. Each ::Beacon:: is of the type
-SingletonBeacon.
+Bitcoin `network` specified by the DID:
+
+* Pay-to-Public-Key-Hash (P2PKH)
+* Pay-to-Witness-Public-Key-Hash (P2WPKH); and
+* Pay-to-Taproot (P2TR).
+
+Spends from these three addresses can be produced only through signatures from the
+private key associated with the `keyBytes`. Each ::BTC1 Beacon:: is of the type
+[SingletonBeacon].
 
 It takes the following inputs:
-* `identifier`: A valid **did:btc1** identifier
-* `keyBytes`: a compressed SEC encoded secp256k1 public key
-* `network`: the bitcoin network used for the identifier
+
+* `identifier` - a valid **did:btc1** identifier; REQUIRED; string
+* `keyBytes` - a compressed SEC encoded secp256k1 public key; REQUIRED; bytes
+* `network` - the Bitcoin network used for the identifier; REQUIRED; string; default=`"bitcoin"`
 
 It returns the following output:
-* `services`: an array of beacon services objects containing the following properties
-   * `type`: a string signaling the type of beacon, default='SingletonBeacon'
-   * `id`: the **did:btc1** identifier controlling the beacon plus the DID fragment pointing
-      to the location of a key in the DID document;
-   * `serviceEndpoint`: a P2PKH, P2WPKH or P2TR bitcoin address where beacon signals can be broadcast
-   ```{.json include="json/CRUD-Operations/Read-deterministically-generate-beacon-services.json"}
-   ```
+
+* `services` - an array of ::BTC1 Beacon:: service objects containing the following properties
+   * `type` - described the kind of service being defined used to determine how to produce updates; 
+      REQUIRED; string; default=`"SingletonBeacon"`
+   * `id` - the **did:btc1** identifier controlling the beacon including the DID fragment pointing
+      to the location of a key in the DID document; REQUIRED; string
+   * `serviceEndpoint` - a P2PKH, P2WPKH or P2TR bitcoin address where beacon signals can be broadcast; REQUIRED; string
```

Comment:
I think this is fine in this PR, but just want to flag that we will need to update this later regarding the new, still slightly debated, type structure for Beacons. i.e. type: ["BTC1Beacon", "SingletonBeacon"]. Or the other variation